app/forms/conversion/create_project_form.rb

```diff
@@ -36,11 +38,17 @@ class NegativeValueError < StandardError; end
   validate :establishment_exists, if: -> { urn.present? }
   validate :trust_exists, if: -> { incoming_trust_ukprn.present? }
 
+  validates :directive_academy_order, inclusion: {in: [true, "true", false, "false"]}
```

Comment:
This is awful but it seems like these are bools or strings depending of whether they're coming from a test or the UI?!

Reply:
I feel like these should be strings in the tests as that is what gets submitted from the form - but it is not a deal breaker - we have the same thing for `assigned_to_regional_caseworker_team`